src/components/NewLocationPage/NotesBlock/NotesBlock.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import NotesBlock from './NotesBlock';
+import VulnerabilityIcon from 'assets/images/VulnerabilityIcon';
+import BaseButton from 'components/ButtonSystem/BaseButton/BaseButton';
+import { EventAction, EventCategory } from 'components/Analytics';
+import { isHighVulnerability } from './utils';
+
+export default {
+  title: 'Location page redesign/Notes Block',
+  component: NotesBlock,
+};
+
+const highVulnerability = isHighVulnerability(1);
+const title = 'Vulnerability is very high';
+const body =
+  'New York City metro is more likely to experience severe physical and economic suffering from COVID, and to face harder, longer recovery.';
+const redirectUrl = 'http://www.google.com';
+
+export const Example = () => {
+  if (!highVulnerability) {
+    return null;
+  }
+  return (
+    <BaseButton
```

Comment:
it looks like `BaseButton` is putting some unwanted styling on the children/NotesBlock. i think we need to make a styled BaseButton that explicitly doesn't style its children. something like:

```
export const StyledBaseButton = styled(BaseButton)`
font-weight: inherit;
font-size: inherit;
text-transform: none;
letter-spacing: 0;
`
```

And then import it into the file and swap `BaseButton` for `StyledBaseButton`